spec/features/18f_procurement_spec.rb

```diff
@@ -27,7 +27,7 @@
       expect(Dispatcher).to receive(:deliver_new_proposal_emails)
 
       visit '/gsa18f/procurements/new'
-      fill_in "Product Name and Description", with: "buying stuff"
+      fill_in "Product name and description", with: "buying stuff"
```

Comment:
Mind if we punt on that? :football: :runner: `t()` doesn't work unless we enter translations for each attribute explicitly, which doesn't seem worth the effort.